modules/javafx.base/src/main/java/javafx/beans/value/ObservableValue.java

```diff
@@ -297,4 +301,48 @@ default <U> ObservableValue<U> flatMap(Function<? super T, ? extends ObservableV
     default ObservableValue<T> when(ObservableValue<Boolean> condition) {
         return new ConditionalBinding<>(this, condition);
     }
+
+    /**
+     * Creates a {@link Subscription} on this value which calls the given
+     * {@code subscriber} with the old and new value of this value whenever it
+     * changes.<p>
+     *
+     * The parameters supplied to the {@link BiConsumer} are the old and new value
+     * respectively.
+     *
+     * @param subscriber a {@code BiConsumer} to supply with the old and new values
+     *     of this {@code ObservableValue}, cannot be {@code null}
+     * @return a {@code Subscription} which can be used to cancel this
+     *     subscription, never {@code null}
+     * @since 21
+     */
+    default Subscription changes(BiConsumer<? super T, ? super T> subscriber) {
+      Objects.requireNonNull(subscriber, "subscriber cannot be null");
+      ChangeListener<T> listener = (obs, old, current) -> subscriber.accept(old, current);
+
+      addListener(listener);
+
+      return () -> removeListener(listener);
+    }
+
+    /**
+     * Creates a {@link Subscription} on this value which immediately provides
+     * the current value to the given {@code subscriber}, followed by any
+     * subsequent changes in value.
+     *
+     * @param subscriber a {@link Consumer} to supply with the values of this
+     *     {@code ObservableValue}, cannot be {@code null}
+     * @return a {@code Subscription} which can be used to cancel this
+     *     subscription, never {@code null}
+     * @since 21
+     */
+    default Subscription values(Consumer<? super T> subscriber) {
```

Comment:
Same question about the names: we might want to consider a name that has "add" or "subscribe" in the name.